armadillo/src/main/resources/application.yml

```diff
@@ -18,11 +21,16 @@ datashield:
           # the seed can only be 9 digits
           seed: 342325352
 #          foo: bar
-    - name: exposome
-      environment: exposome
+#    - name: exposome
+#      environment: exposome
+#      whitelist:
+#        - dsBase
+#        - dsExposome
+    - name: omics
```

Comment:
Should we comment out the `omics` profile too?

Reply:
Yes, it needs to be done in de docker-compose as well. I will do this.